src/Gaffer/Reference.cpp

```diff
@@ -172,6 +174,15 @@ void Reference::loadInternal( const std::string &fileName )
 				convertPersistentMetadata( it->get() );
 			}
 		}
+		else if( Node *node = runTimeCast<Node>( newChildren->member( i ) ) )
```

Comment:
I do. Load time is something I'd like to improve on, but Python is our main enemy there. The `member( i )` method is a constant-time lookup since the `StandardSet` internally uses a `multi_index_container` so it can do both quick membership queries and also quick lookup by index. I can definitely stick it in a temporary though if you would prefer - I am probably guilty of the habit of repeating this sort of thing unnecessarily.